src/Features/CSharp/Portable/ChangeSignature/CSharpChangeSignatureService.cs

```diff
@@ -171,7 +172,7 @@ protected override int GetPositionBeforeParameterListClosingBrace(SyntaxNode mat
             {
                 ParameterListSyntax parameterListSyntax => parameterListSyntax.CloseParenToken.SpanStart,
                 BracketedParameterListSyntax bracketedParameterListSyntax => bracketedParameterListSyntax.CloseBracketToken.SpanStart,
-                _ => throw new ArgumentException("Unexpected SyntaxNode", nameof(matchingNode))
+                _ => matchingNode.SpanStart
```

Comment:
why?

Reply:
It was commented in master but not here. This case is for unparenthesized lambda parameters.